app/MMB/src/scenes/appleWallet/AppleWalletScene.js

```diff
@@ -0,0 +1,31 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Color } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+import HeaderImage from './HeaderImage';
+
+export default function AppleWalletScene() {
+  return (
+    <React.Fragment>
+      <HeaderImage />
+      <View style={styles.content}>
+        <Translation passThrough="TODO" />
+      </View>
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+  content: {
+    flex: 1,
+    backgroundColor: Color.white,
+    borderTopStartRadius: 10,
+    borderTopEndRadius: 10,
```

Comment:
Does `borderTopHorizontalRadius` exist by any chance? 😄

Reply:
I wish it did 😞

Reply:
Or just borderTopRadius would also be fine 😄